### PR TITLE
fix package builds in some cases

### LINKED ISSUE #1991: pkg/init fails to build
```
~/packages/linuxkit/pkg/init master make
docker build --no-cache --network=none -t linuxkit/init:83f70b224a300e63f8df62d0917486d5e8b8bb11 .
Sending build context to Docker daemon  17.92kB
Step 1/12 : FROM linuxkit/containerd:5749f2e9e65395cc6635229e8da0e0d484320ddf AS mirror
 ---> 27a76c5adbb3
Step 2/12 : RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
 ---> Running in a0ce82bb2b2e
container_linux.go:262: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory"
oci runtime error: container_linux.go:262: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory"

Makefile:11: recipe for target 'tag' failed
make: *** [tag] Error 1
```

Seems like we're using containerd's image as our source image, which itself is built from a scratch image. I can switch it back to linuxkit/alpine, but I assume that's not what we want here. I'm happy to do the PR if someone knows the right fix.

### PR DESCRIPTION
**Pull request by tych0**
0eb21735ae1b73d6012beff2898aaae49bf46f34 accidentally broke some package
builds by switching linuxkit/alpine to linuxkit/containerd. Let's revert
the ones that shouldn't be there.

Closes #1991

Signed-off-by: Tycho Andersen <tycho@docker.com>

### CODE REVIEW

File: pkg/rngd/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM linuxkit/containerd:5749f2e9e65395cc6635229e8da0e0d484320ddf AS mirror
+FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS mirror
 RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
 RUN apk add --no-cache --initdb -p /out \
```
Comment (riyazdf) [inline pkg/rngd/Dockerfile]: Wait - there's another containerd that needs to change on the second part of this build

Comment (tych0) [inline pkg/rngd/Dockerfile]: Indeed, thanks!

Comment (riyazdf) [inline pkg/rngd/Dockerfile]: 👍

**Comment by riyazdf** [comment]
I've verified this locally, thanks @tych0!  I'm debugging why https://github.com/linuxkit/linuxkit/pull/1992 is green (it shouldn't be) but will merge this since this is the proper fix.

**Pull request merged**